Match IDP templates to main template

RubenVerborgh opened the pull request:
Closes #772

RubenVerborgh commented:
@joachimvh I've done 3 pages as and example, is this sufficient to help you on your way?
I wasn't sure how to trigger some of the other pages.

When this is done, we can delete the IDP CSS.

Also wonder to what extent the IDP pages could reuse `templates/main.html` (as I'm now doing manual copies).

joachimvh commented:
I have updated the remaining templates and updated the integration test to work with the new templates.
Also discovered we had 2 unused templates.

Going through it, I realized there's probably an issue with the template used when sending an e-mail (`message.ejs`). It's linking to the stylesheet, but I'm guessing that's probably not going to be found when received as an e-mail?

I like the update to the registration page, makes it a bit clearer. The password fields are also only needed when registering with the IDP (so 2nd checkbox) should you have an idea on how to make that clearer.

RubenVerborgh commented:
> I have updated the remaining templates and updated the integration test to work with the new templates.

Thanks. You will need to review though, given that I started the PR.

> It's linking to the stylesheet, but I'm guessing that's probably not going to be found when received as an e-mail?

Can we add the hostname?

> The password fields are also only needed when registering with the IDP (so 2nd checkbox) should you have an idea on how to make that clearer.

Yeah, I might update those with some client-side scripting at a later point in time.

RubenVerborgh commented:
Let's remove the IDP CSS and any related assets if there are some (images etc.).

joachimvh reviewed (approved):
I was wrong about the mail stylesheet so that comment can be ignored.
